tests/foreman/cli/test_satellitesync.py

```diff
@@ -64,13 +64,58 @@ def config_export_import_settings():
     Settings.set({'name': 'content_disconnected', 'value': disconnected_mode_value})
 
 
+@pytest.fixture(scope='function')
+def export_cleanup_function(default_sat, function_org):
+    """Deletes export/import dirs of function org"""
+    yield
```

Comment:
Does the empty `yield` just ensure the teardown is performed after the test is finished?